straxen/corrections_services.py

```diff
@@ -19,15 +19,11 @@
     "cnn_model",
     "gcn_model",
     "s2_xy_map_mlp",
-    "s2_xy_map_cnn",
```

Comment:
I think you still need to keep these. Because the remaining MLP and CNN plugins need them.

Reply:
You are right as we probably still need to calculate those plugins at some point. Including them there doesn't harm as those maps are not necessarily called during the processing. I added them back.